frontend/src/components/MapView/Layers/AnalysisLayer/index.tsx

```diff
@@ -77,7 +79,22 @@ const onClick = (analysisData: AnalysisResult | undefined) => ({
     );
     dispatch(
       addPopupData({
-        [analysisData.getStatTitle(t)]: {
+        [t('Analysis layer')]: {
+          data: analysisData.getLayerTitle(t),
+          coordinates,
+        },
+        ...(analysisData.analysisDate
```

Comment:
Would be end up adding popupData if there was no analysisDate?

Reply:
I'm not sure to be honest. According to the type definitions, analysisDate is nullable on BaselineLayerResult but I'm not sure when/how/if that's actually possible
<img width="404" alt="Screenshot 2024-06-19 at 8 17 11 AM" src="https://github.com/WFP-VAM/prism-app/assets/8203830/ea989a78-e832-46ca-b21e-5613338fabb7">

Reply:
I guess in theory we could analyze a layer that is independent of time so let's live it like that